neurovault/apps/statmaps/views.py

```diff
@@ -1224,7 +1224,7 @@ def download_collection(request, cid):
         zf.write(fpath, zip_path)
 
     response = StreamingHttpResponse(zf, content_type='application/zip')
-    response['Content-Disposition'] = 'attachment; filename=%s' % zip_filename
+    response['Content-Disposition'] = 'attachment; filename=%s' % zip_filename.replace (" ", "_")
```

Comment:
`urllib.quote_plus(zip_filename)` gives `+` instead of spaces and
`urllib.pathname2url(zip_filename)` gives `%20` instead of spaces.

I will use the first one ¯\_(ツ)_/¯